packages/node_modules/pouchdb-adapter-utils/src/parseDoc.js

```diff
@@ -76,7 +76,12 @@ function makeRevTreeFromRevisions(revisions, opts) {
 
 // Preprocess documents, parse their revisions, assign an id and a
 // revision for new writes that are missing them, etc
-function parseDoc(doc, newEdits) {
+function parseDoc(doc, newEdits, dbOpts) {
+  if (!dbOpts) {
+    dbOpts = {
+      deterministic_rev: true
```

Comment:
Sorry if I got it wrong in previous comment, I think it should be `determinstic_revs` here